### PR TITLE
Tips page

### PR DESCRIPTION
**Pull request by T0ny-X**
image
image

### CODE REVIEW

**Comment by T0ny-X** [comment]
https://magic-kitchen-b5f37--pr15-tips-page-1oad9e96.web.app/tips

File: src/routes/tips.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { BlogCard } from "../components/Card"
+import Layout from "../components/Layout";
+import FaqSection from "../components/FaqSection"
+import { FaqOptions } from "../data/faq.ts"
+function TipsList() {
+    return (
+        <Layout>
+            <Box my={8} textAlign="center">
+              <Heading margin="5vh">Tips</Heading>
+                <Box display="grid" gridGap="4" gridTemplateColumns="repeat(auto-fit, minmax(300px, 1fr))">
```
Comment (hetd54) [inline src/routes/tips.jsx:13]: can we center cards on mobile?

**Review by hetd54 — CHANGES_REQUESTED** [review]

**Review by AD121390 — APPROVED** [review]
Looks great. My only nit pick is that the main title is the only clickable link. It would be nice if the entire card or picture was also a clickable link, but not a big deal.

**Review by hetd54 — APPROVED** [review]
Amazing 🚀

**Pull request merged**